### DISCUSSION TITLE
Zod integration

### DISCUSSION THREAD

**Discussion by CapitaineToinon** [vueuse/vueuse#2546] [Ideas]
I've been looking at ways to intergrate my tRPC zod schemas with my Nuxt 3 frontend and I was really surprised to find out that there isn't any nice solution for vue. vee-validate claims to have zod support [but the typescript support is pretty awful](https://github.com/logaretm/vee-validate/issues/3375). 

In th react world they seem to have plently of solutions but nothing pretty clean in Vue. I think vueuse could be a good candiate for that, to have a simple `useZod` validator. Thoughts?

**Comment by prpanto** [comment]
Check [h3-zod](https://github.com/wobsoriano/h3-zod)
